app/pages/multisig/updates/CreatePLT/CreatePltParameters.tsx

```diff
@@ -0,0 +1,216 @@
+import React from 'react';
+
+import { TokenHolder } from '@concordium/web-sdk/plt';
+import { AccountAddress } from '@concordium/web-sdk';
+
+import { EqualRecord } from '~/utils/types';
+import Form from '~/components/Form/';
+import { mustBeAnInteger, requiredMessage } from '../common/util';
+
+export interface UpdateCreatePltParametersFields {
+    tokenId: string;
+    name: string;
+    moduleRef: string;
+    metadataUrl: string;
+    metadataHash?: string;
+    governanceAccount: string;
+    decimals: number;
+    initialSupply?: bigint;
+    allowList?: boolean;
+    denyList?: boolean;
+    mintable?: boolean;
+    burnable?: boolean;
+}
+
+const fieldNames: EqualRecord<Required<UpdateCreatePltParametersFields>> = {
+    tokenId: 'tokenId',
+    name: 'name',
+    moduleRef: 'moduleRef',
+    metadataUrl: 'metadataUrl',
+    metadataHash: 'metadataHash',
+    governanceAccount: 'governanceAccount',
+    decimals: 'decimals',
+    initialSupply: 'initialSupply',
+    allowList: 'allowList',
+    denyList: 'denyList',
+    mintable: 'mintable',
+    burnable: 'burnable',
+};
+
+export const fieldDisplays = {
+    tokenId: 'Token ID',
+    name: 'Name',
+    moduleRef: 'Module Reference',
+    metadataUrl: 'Metadata Url',
+    metadataHash: 'Metadata Hash',
+    governanceAccount: 'Governance Account',
+    decimals: 'Decimals',
+    initialSupply: 'Initial Supply',
+    allowList: 'Allow List',
+    denyList: 'Deny List',
+    mintable: 'Mintable',
+    burnable: 'Burnable',
+};
+
+/**
+ * Component for creating an update create PLT transaction.
+ */
+export default function CreatePltParameters(): JSX.Element | null {
+    return (
+        <div>
+            <Form.Input
+                className="body2 mB20"
+                name={fieldNames.tokenId}
+                label={`${fieldDisplays.tokenId}`}
+                placeholder="Alphanumeric and . % - (max 128 chars)"
+                maxLength={128}
+                pattern="^[a-zA-Z0-9.%-]{1,128}$"
+                rules={{
+                    required: requiredMessage(fieldDisplays.tokenId),
+                    validate: (value: string) => {
+                        const cleaned = value.replace(/[^a-zA-Z0-9.%-]/g, '');
+                        if (cleaned.length > 128) {
+                            return 'Must be 128 characters or less.';
+                        }
+                        if (value !== cleaned) {
+                            return "Only letters, numbers, '.', '%', and '-' are allowed.";
+                        }
```

Comment:
I would flip these:

```suggestion
                        if (value !== cleaned) {
                            return "Only letters, numbers, '.', '%', and '-' are allowed.";
                        }
                        if (cleaned.length > 128) {
                            return 'Must be 128 characters or less.';
                        }
```

otherwise you're checking the length the value that is not in the input field, which seems weird to me